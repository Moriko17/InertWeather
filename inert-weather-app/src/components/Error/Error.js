import React from "react";

export default function(props) {
  return (
    <div className="error">
      <p>Something goes wrong: {props.errorMessage}</p>
    </div>
  );
}
