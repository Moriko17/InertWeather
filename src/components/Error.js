import React from "react";

export default function() {
  return (
    <div className="error-main">
      <p>Error</p>
    </div>
  );
}
