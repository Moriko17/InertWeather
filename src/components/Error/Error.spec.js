import React from "react";
import renderer from "react-test-renderer";
import Error from "./Error";

describe("AddToFavorite renders right", () => {
  it("Renders correctly", () => {
    const tree = renderer.create(<Error errorMessage={404} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
