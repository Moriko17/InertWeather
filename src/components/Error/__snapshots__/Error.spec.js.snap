// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddToFavorite renders right Renders correctly 1`] = `
<div
  className="error"
>
  <p>
    Something goes wrong: 
    404
  </p>
</div>
`;
